modules/bmo-about-the-bare-metal-operator.adoc

```diff
@@ -0,0 +1,69 @@
+
+The BMO interfaces with bare-metal hosts in the cluster by using the Ironic API service. The Ironic service uses the Baseboard Management Controller (BMC) on the host to interface with the machine. The BMO continually inspects the bare-metal hosts in your cluster to ensure each `BareMetalHost` resource accurately reflects the hardware, software, and firmware details for each bare-metal host in the cluster. 
+
+The BMO also uses the `HostFirmwareSettings` resource and the `FirmwareSchema` resource to detail firmware specifications for the bare-metal host. 
+
+Some common tasks you can complete by using the BMO include the following:
+
+* Provision bare-metal hosts to the cluster with a specific image
+* Format a host's disk contents before provisioning or after deprovisioning
+* Turn on or off a host
+* Change firmware settings
+* View the host's hardware details 
+
+DRAFT IMAGE: 
+
+image::BMO-Diagram-v2.png[DRAFT - PLACEHOLDER IMAGE]
+
+.BareMetalHost
+
+The `BareMetalHost` resource defines a physical host and its properties. When you provision a bare-metal host to the cluster, you must define a `BareMetalHost` resource for that host. For ongoing management of the host, you can inspect the information in the `BareMetalHost` or update this information. 
+
+The `BareMetalHost` resource features provisioning information such as the following:
+
+* URL of the operating system boot image
+* Provisioning state 
+* Baseboard Management Controller (BMC) address
+* Desired power state
+
+The `BareMetalHost` resource features hardware information such as the following: 
+
+* Number of CPUs
+* MAC address of a NIC
+* Size of the host's storage device
+* Power state
+
+.HostFirmwareSettings
+You can use the `HostFirmwareSettings` resource to retrieve and manage the firmware settings for a host. When a host moves to the `Available` state, the Ironic service reads the host's firmware settings and creates the `HostFirmwareSettings` resource. There is a one-to-one mapping between the `BareMetalHost` resource and the `HostFirmwareSettings` resource. 
+
+You can use the `HostFirmwareSettings` resource to inspect the firmware specifications for a host or to update a host's firmware specifications. 
+
+[NOTE]
+====
+You must adhere to the schema specific to the vendor firmware when you edit the `spec` field of the `HostFirmwareSettings` resource. This schema is defined in the read-only `FirmwareSchema` resource.
+====
+
+.FirmwareSchema
+Firmware settings vary among hardware vendors and host models. A `FirmwareSchema` resource is a read-only resource that contains the types and limits for each firmware setting on each host model. The data comes directly from the BMC by using the Ironic service. The `FirmwareSchema` resource enables you to identify valid values you can specify in the `spec` field of the `HostFirmwareSettings` resource. 
```

Comment:
@dtantsur - is there a 1-1 or 1-many relationship between the `FirmwareSchema` resource and BareMetalHost resources? In other words, is one `FirmwareSchema` resource used by many BMH resources? I might clarify this in this paragraph, unless you think it's not needed

Reply:
FirmwareSchema extracted for each host independently (and may be different).